database/database_test.go

```diff
@@ -0,0 +1,51 @@
+package database
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/moov-io/base/docker"
+)
+
+func Test_NewAndMigration_SqlLite3(t *testing.T) {
+	_, close, err := NewAndMigrate(InMemorySqliteConfig, nil, nil)
+	if err != nil {
+		t.FailNow()
+	}
+	close()
+}
+
+func Test_NewAndMigration_MySql(t *testing.T) {
+	if !docker.Enabled() {
+		t.SkipNow()
+	}
+
+	config := DatabaseConfig{
+		DatabaseName: "identity",
+		MySql: &MySqlConfig{
+			Address:  "tcp(localhost:4306)",
+			User:     "identity",
+			Password: "identity",
+		},
+	}
+
+	_, close, err := NewAndMigrate(config, nil, nil)
```

Comment:
I'm curious how does it work if we don't have mysql instance running and we do not expect error here?

Reply:
I missed your comment on this.